automation_testing/tests/FormFields/FormFieldNumberTable/FormFieldNumberTable.spec.ts

```diff
@@ -72,6 +72,15 @@ test.describe.parallel("FormFields - FormFieldNumberTable - Playground", () => {
 		expect(await ffNumberTablePage.numberTableLocator.locator("thead").locator("th").last().textContent()).not.toBe("No. Rooms");
```

Comment:
Got several errors in this file. It's expecting Not a Number, which I assume might be happening because it's reading the numbers as prices (e.g. "$123.00") and then tries to add them with the "$" which causes it to break. I think you might be able to solve this using something like 
inputSum += Number(await this.tableBodyRowLocator.nth(j).locator("input").nth(i).inputValue()**_.split("$")[1]_** );

![image](https://user-images.githubusercontent.com/23409284/236566581-f7c74609-8e51-4fdb-a3be-846bbf5c2c3d.png)
![image](https://user-images.githubusercontent.com/23409284/236566625-29378d48-f767-41af-ab48-8bf7929c0a5d.png)
![image](https://user-images.githubusercontent.com/23409284/236566652-e0bb676f-9081-4b99-ad04-2674b737fffe.png)
![image](https://user-images.githubusercontent.com/23409284/236566778-2169272a-06da-444d-8422-879c52543ab9.png)
![image](https://user-images.githubusercontent.com/23409284/236566800-df066a3b-0559-44a8-9a77-a2d0684900f2.png)